dispatcher/backend/src/common/schemas/offliners/kolibri.py

```diff
@@ -49,6 +50,16 @@ class Meta:
         }
     )
 
+    long_description = fields.String(
```

Comment:
We should probably open a ticket to switch all long-description fields to a textarea with 4K chars limit

Reply:
I do not see any long_description field in any other scraper. 

I suggest that we create a ticket to:
- move the logic regarding truncating / defaults of description / long_description that we developed in `kolibri` to `pythonscraperlib` so that it is shared across scrapers
- add support for both description / long_description CLI flags in all python scrapers
- add support for textarea in the UI for flags fields
- limit field maximum length in the UI (I'm not sure about this one, is there any validation logic implemented in the UI ? maybe sufficient to do this in the backend when validating the payload received)

WDYT?

Reply:
> I do not see any long_description field in any other scraper.

Not sure if it's in zimfarm but I remember it being supported elsewhere (nautilus at least). `mwoffliner` (not python) does support it in the zimfarm.
 
> I suggest that we create multiple issues to:
> 
>     * move the logic regarding truncating / defaults of description / long_description that we developed in `kolibri` to `pythonscraperlib` so that it is shared across scrapers

Should support both cases: with or without a potential long description.

>     * add support for both description / long_description CLI flags in all python scrapers

Description is already everywhere ; but needs to be updated. Would require updating scraperlib to latest version.

>     * add support for textarea in the UI for flags fields

👍 

>     * limit field maximum length in the UI (I'm not sure about this one, is there any validation logic implemented in the UI ? maybe sufficient to do this in the backend when validating the payload received)

We'd check in the backend for sure. For frontend, textarea/input's `maxlength` is good enough.

Reply:
Opened :
- https://github.com/openzim/zimfarm/issues/810
- https://github.com/openzim/python-scraperlib/issues/110
- https://github.com/openzim/overview/issues/21